pkg/agent/util/net_linux.go

```diff
@@ -240,6 +241,34 @@ func DeleteOVSPort(brName, portName string) error {
 	return cmd.Run()
 }
 
+func HostInterfaceExists(ifName string) bool {
+	_, err := netlink.LinkByName(ifName)
+	if err == nil {
+		return true
+	}
+	if _, ok := err.(netlink.LinkNotFoundError); ok {
+		return false
+	}
+	klog.ErrorS(err, "Failed to find host interface", "name", ifName)
```

Comment:
why log an error? that doesn't make sense for this function. If the caller needs the interface to exist, then the caller should be the one logging the error when it is missing

Reply:
@mengdie-song 
Can you please take care of this in your patch?